### ISSUE TITLE
Invalid aria-relevent attribute added to Pagination aria-live component

### ISSUE THREAD

**Issue by alanbsmith** [Workday/canvas-kit#1658]
## 🐛 Bug Report

An invalid `aria-relevent` attribute was erroneously added to Pagination aria-live component. It doesn't appear to be harming anything, but it also isn't needed and should be removed.

## Link to repl or repo (highly encouraged)

Remove this line and the associated test.
https://github.com/Workday/canvas-kit/blob/master/modules/react/pagination/lib/Pagination/common/useLiveRegion.tsx#L15
